pallets/randomness-beacon/src/benchmarking.rs

```diff
@@ -14,68 +14,103 @@
  * limitations under the License.
  */
 
-//! Benchmarking setup for pallet-drand
+//! Benchmarking setup for pallet-randomness-beacon
+#![cfg(feature = "runtime-benchmarks")]
 use super::*;
 
 #[allow(unused)]
-use crate::{pallet as pallet_drand, Pallet as Drand};
-use ark_ec::Group;
-use ark_std::{ops::Mul, UniformRand};
-use frame_benchmarking::v2::*;
-use frame_support::BoundedVec;
+use crate::{pallet as pallet_randomness_beacon, Pallet};
+use ark_bls12_381::G1Affine as G1AffineOpt;
```

Comment:
should we feature flag "host-arkworks" here?
This will help us benchmark how much better is to use the host functions. Also, it won't penalize callers when the chain finally gets the host functions (if that ever happens)
```suggestion
#[cfg(not(feature = "host-arkworks"))]
use ark_bls12_381::G1Affine as G1AffineOpt;
#[cfg(feature = "host-arkworks")]
use sp_ark_bls12_381::::G1Affine as G1AffineOpt;
```